### ISSUE TITLE
ssh hangs on centos7 version 1.1.4

### ISSUE THREAD

**Issue by cdmo** [geerlingguy/drupal-vm#960]
Tried to start a new DrupalVM box up using geerlingguy/centos7 1.1.4 and kept getting hung at the point where ssh logins occur. No error messages would show either, it would just sort of hang for a while and then exit out saying it couldn't get ssh access. I am assuming this is a problem with the 1.1.4 release because when I made a `Vagrantfile.local` and added the following line, destroyed and ran up again, it was able to execute ssh.

`config.vm.box_version = "1.1.3"`

Maybe just me?

**Comment by paulsheldrake** [comment]
I have the same issue

**Comment by geerlingguy** [comment]
Grr, I thought I had it working. I definitely ran tests on the base box before pushing it up, and they passed with no Ansible configuration, maybe some SSH setting is messed up.

Could you post an issue here? https://github.com/geerlingguy/packer-centos-7/issues

**Comment by cdmo** [comment]
It's funny, I almost did post the issue there, just wasn't sure which queue was right, lol. Thanks.

**Comment by geerlingguy** [comment]
@cdmo / @paulsheldrake - Can you check what version of Vagrant and VirtualBox you're running? The networking issue was fixed in 5.1.6, but I built the latest box for 5.1.8 and 1.8.6 of Vagrant.

**Comment by cdmo** [comment]
vagrant 1.8.5
virtualbox 5.0.28

note, I can't upgrade to Vagrant 1.8.6 due to #912

**Comment by paulsheldrake** [comment]
vagrant 1.8.6
virtualbox 5.1.8

i did the upgrades to fix it but it still fails

**Comment by geerlingguy** [comment]
@paulsheldrake - After doing so, you'll also need to downgrade the box, then after that and a provision, upgrade the box again, due to Vagrant/VirtualBox caching and cloning behavior.

**Comment by geerlingguy** [comment]
Closing out this issue, as the main problem is an older version of VirtualBox, and that + getting its caches cleared (by uninstalling and reinstalling the box or downgrading and upgrading) should solve this particular issue.

#912 is a separate issue, and will unblock the ability for some people to upgrade, but this particular issue can be closed independently, since it's fixed with the latest versions.

**Comment by cdmo** [comment]
I can say it isn't working for me still

vagrant 1.8.7
VirtualBox 5.1.6 (can't use 5.1.8 due to geerlingguy/drupal-vm#943

Ran the following to remove the newest version:

`vagrant box remove geerlingguy/centos7 --box-version=1.1.4`

Ran `vagrant box update` to reinstall the new version and got the following:

```
==> drupalvm: Checking for updates to 'geerlingguy/centos7'
    drupalvm: Latest installed version: 1.1.3
    drupalvm: Version constraints:
    drupalvm: Provider: virtualbox
There was an error while downloading the metadata for this box.
The error message is shown below:
```

Except there is no error message shown below...

Did I do something wrong in my uninstall/reinstall process?

**Comment by geerlingguy** [comment]
@cdmo - Please use Vagrant 1.8.6 instead (or use the linked issue's workaround). 1.8.7 has a major bug: https://github.com/mitchellh/vagrant/issues/7969

**Comment by cdmo** [comment]
Did fix! Thanks @geerlingguy ✅
